pkg/tlsf/.gitattributes

```diff
@@ -1,2 +0,0 @@
-# Ignore whitespace errors inside these patch files by regarding them as binary.
-patch.txt binary
```

Comment:
> Why not?

Because with a seperate repository I can only review the most current state of the patch files. With the patch files not being binary I can review the history of the patch files themselves (I mean locally on my computer (or alternatively in the GitHub UI), not in the PR reviewing process).